sejima-library/app/src/main/java/com/rlab/sejima/features/MUPinCode.java

```diff
@@ -0,0 +1,390 @@
+package com.rlab.sejima.features;
```

Comment:
So UI reports.

- When I write, I can write as many characters as I want, should be see next point.
- When you enter a character, you should change to next PIN label (to avoid manually clicking).
- Is it possible to hide the cursor, to hide it is a Textfield?
- It will look nicer with the character "•" instead of the "." dot (more center and bigger)?
- If I have 6 pin code, on the 3 first I can discard the keyboard with the return key, not on the 3 others.

Reply:
all points are fixed